src/pages/dex/index.page.tsx

```diff
@@ -109,20 +109,20 @@ export default function DexPage({
 }
 
 export async function getServerSideProps(
-  context: GetServerSidePropsContext
+  context: GetServerSidePropsContext,
 ): Promise<GetServerSidePropsResult<DexPageProps>> {
   const api = getWhaleApiClient(context);
 
   const next = CursorPagination.getNext(context);
   const items = await api.poolpairs.list(100, next);
   const sorted = items.filter(
-    (poolpair) => !poolpair.displaySymbol.includes("/")
+    (poolpair) => !poolpair.displaySymbol.includes("/"),
   );
 
   return {
     props: {
       poolPairs: {
-        items: sorted,
+        items: sorted.filter((item) => !item.symbol.includes("BURN")),
```

Comment:
why not include the logic on line 119? less loops

Reply:
overlooked D: